src/components/detail/detail-test.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { ICONS } from "../icon/icon-config";
+import Detail, { DetailProps } from ".";
+
+export default {
+  component: Detail,
+  title: "Detail/Test",
+  includeStories: "Default",
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disableSnapshot: true,
+    },
+  },
+  argTypes: {
+    icon: {
+      options: ICONS,
+      control: {
+        type: "select",
+      },
+    },
+  },
```

Comment:
Since `children` is typed as a node, the storybook control for editing the `children` prop has been automatically set to accept JSON input - since the prop could be a React node in addition to a string. If you press the `a` key in Canvas mode to open the addons sidebar, you'll see this: 

![Screenshot 2023-03-17 at 16 40 10](https://user-images.githubusercontent.com/18368713/225965890-7a54efbb-db6a-444f-a81c-fd4ae946d0af.png)

Since we are only interested in checking if some value passed to `children` renders properly, we can manually change the control input for `children` like so:

```suggestion
  },
  children: {
    control: { type: "text" },
  },
```

This will change the control type to accept text input instead